dashboard/app/controllers/congrats_controller.rb

```diff
@@ -2,6 +2,9 @@ class CongratsController < ApplicationController
   def index
     view_options(full_width: true, responsive_content: true, has_i18n: true)
 
+    # Select two different donors, because the first must have a twitter
+    # handle and the second must be equally weighted across all donors.
     @random_donor_twitter = CdoDonor.get_random_donor_twitter
+    @random_donor_name = CdoDonor.get_random_donor_name
```

Comment:
I'm curious how this works. If the user shares on twitter is a different donor tagged than the one represented on the certificate?

Reply:
correct. I chose this because I believe it gives us parity with previously existing functionality. in the future it should be easy to get rid of get_random_donor_twitter and just post the donor name instead if there is no twitter handle, if we think that's better behavior. 

what do you think about this potential change @tess323 ? I could do this as a follow-up PR if we think that's what we want. in case you missed it, here is the relevant code comment:
```
    # Select two different donors, because the first must have a twitter
    # handle and the second must be equally weighted across all donors.
```

Reply:
I'll make a note to follow up about this with Tess since she is out today.